src/main/java/HomeWork_2/Array.java

```diff
@@ -0,0 +1,127 @@
+package HomeWork_2;
+
+public class Array {
+
+    int arr[];
+
+    public Array(int[] arr) {
+        this.arr = arr;
+    }
+
+    public Array() {
+    }
+
+    public double averageValue(){
+        double sumArray=0;
+        try{
+            for (int i = 0; i<arr.length; i++) {
+                sumArray += arr[i];
+            }
+            sumArray /= arr.length;
+        }catch (NullPointerException ex){
+            System.out.print("Empty array ");
+            ex.printStackTrace(System.out);
+            System.exit(1);
+        }
+        return sumArray;
+    }
+
+    public int minValue(){
+        int min = 0 ;
+        try{
+            min=arr[0];
+            for(int i = 0; i<arr.length; i++){
+                if (min>=arr[i]){
```

Comment:
`(min>arr[i])` would be better, to avoid unnecessary assignment

Reply:
fixed